google-cloud-spanner/test/helper.rb

```diff
@@ -180,18 +180,38 @@ def paged_enum_struct response
   def expect_execute_streaming_sql results_enum, session_name, sql,
                                    transaction: nil, params: nil, param_types: nil,
                                    resume_token: nil, partition_token: nil, seqno: nil,
-                                   query_options: nil, options: nil
+                                   query_options: nil, request_options: nil, options: nil
     spanner.service.mocked_service.expect :execute_streaming_sql, results_enum do |request, gapic_options|
       request[:session] == session_name &&
-        request[:sql] == sql &&
-        request[:transaction] == transaction &&
-        request[:params] == params &&
-        request[:param_types] == param_types &&
-        request[:resume_token] == resume_token &&
-        request[:partition_token] == partition_token &&
-        request[:seqno] == seqno &&
-        gapic_options == options &&
-        request[:query_options] == query_options
+      request[:sql] == sql &&
+      request[:transaction] == transaction &&
+      request[:params] == params &&
+      request[:param_types] == param_types &&
+      request[:resume_token] == resume_token &&
+      request[:partition_token] == partition_token &&
+      request[:seqno] == seqno &&
+      request[:query_options] == query_options &&
+      request[:request_options] == request_options &&
+      gapic_options == options
+    end
+  end
+
+  def expect_execute_update_sql results_enum, session_name, sql,
```

Comment:
This is not used anywhere. Should we delete this method?

Reply:
ok removing it.